Svenbjorn/whatson
# Improvement/maps

3 monday items:
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956373798
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956374404
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956375282

TO TEST:
Verify that the map view in whatson events, places and hiking matches the ones in invision and from figma
